examples/common/python/utility/signature.py

```diff
@@ -177,23 +177,28 @@ def calculate_datahash(self, data_objects):
     def generate_signature(self, hash, private_key):
```

Comment:
generate_signature() is used in work_order_params as well, changes are required there as well